extensions/libraries/redcore/form/base.php

```diff
@@ -73,80 +74,6 @@ public static function getInstance($name, $data = null, $options = array(), $rep
 	}
 
 	/**
-	 * Override the validate() method to store the error per field.
-	 *
-	 * @param   array   $data   An array of field values to validate.
-	 * @param   string  $group  The optional dot-separated form group path on which to filter the
-	 *                          fields to be validated.
-	 *
-	 * @return  mixed  True on sucess.
-	 */
-	public function validate($data, $group = null)
```

Comment:
nvm, I returned it so we have group error support for errors...
this should also go in Joomla base...
thanks @phproberto